drivers/SmartThings/matter-lock/src/init.lua

```diff
@@ -21,6 +21,7 @@ local PowerSource = clusters.PowerSource
 local capabilities = require "st.capabilities"
 local im = require "st.matter.interaction_model"
 local lock_utils = require "lock_utils"
+local COTA_READ_INITIALIZED = "__cota_read_initialized"
```

Comment:
nit: Could we move it to `lock_utils.lua` just to be consistent with other keys, even though temporarily allocated?

Reply:
Fixed in [7c2a912](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1495/commits/7c2a912bb82a0df160e17df5781c62fc3743a4b4)!